CONTRIBUTING.md

```diff
@@ -0,0 +1,95 @@
+# Contributing to NuGetPackageChecker
+
+All contributions are encouraged and valued. Please read the [Table of Contents](#table-of-contents) before contributing to help maintainers and improve the experience for everyone. The community looks forward to your contributions.
```

Comment:
I think this should reference and link to the Code of Conduct instead. Maybe "Please read the Code of Conduct in addition to this contributing guide before contributing... "